Woof/Woof/Model/Walking/Walking+Dummy.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+extension Walking {
+    /// Provides dummy data for `Walking` model.
+    enum Dummy {
+        static let dummyWalking = Walking(
+            id: UUID(),
+            owner: Owner.Dummy.laraCroft,
+            sitter: Sitter.Dummy.emilyDoe,
+            status: .available,
+            start: Date(),
+            end: Date() + 2400,
+            location: Location(longitude: 0.0, latitude: 0.0, address: "Sample Address"),
+            reviews: Reviews(ownerReview: "", sitterReview: ""),
+            rating: Ratings(ownerRating: 0, sitterRating: 0),
+            notes: "Sample notes",
+            price: 28.0 // You can set the price as needed
+        )
+
+        static let bulkDummyWalkings = [
```

Comment:
Missing doc

Reply:
In my opinion, you could understand the purpose of these constants from their names.
Fixed with 7f70e76